Wrappers/Python/test/test_functions.py

```diff
@@ -326,6 +326,7 @@ def test_Norm2sq_as_FunctionOperatorComposition(self):
         
         func1 = FunctionOperatorComposition(0.5 * L2NormSquared(b = b), operator)
         func2 = LeastSquares(operator, b, 0.5)
+        print("func1.L {}, func2.L {}, operator norm {}".format(func1.L, func2.L, operator.norm()))
```

Comment:
what  is this?

Reply:
what do you mean? It's a print in the unittest